components/transfer/TransferLocale.cs

```diff
@@ -3,9 +3,9 @@
     public class TransferLocale
     {
         public string NotFoundContent { get; set; }
-        public string SearchPlaceholder { get; set; }
-        public string ItemUnit { get; set; }
-        public string ItemsUnit { get; set; }
+        public string SearchPlaceholder { get; set; } = "Search here";
+        public string ItemUnit { get; set; } = "item";
+        public string ItemsUnit { get; set; } = "items";
         public string Remove { get; set; }
```

Comment:
Missing?

Reply:
https://github.com/ant-design-blazor/ant-design-blazor/blob/8f128a878e8f324abeb481cbd89932333a98a74f/components/locales/en-US.json#L171-176

There are only three properties in the json file.

Reply:
And other properties are not used.